fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java

```diff
@@ -67,15 +66,18 @@ public ResponseEntity listAccounts(SessionEntity sessionEntity,
     }
 
     private ResponseEntity readOpbaResponse(String bankID, SessionEntity sessionEntity, String redirectCode) {
+        UUID xRequestId = UUID.fromString(restRequestContext.getRequestId());
         ResponseEntity accounts;
         if (null != sessionEntity.getServiceSessionId() && sessionEntity.getConsentConfirmed()) {
             accounts = tppAisClient.getAccounts(
-                    tppProperties.getFintechID(),
                     tppProperties.getServiceSessionPassword(),
                     sessionEntity.getLoginUserName(),
                     RedirectUrlsEntity.buildOkUrl(uiConfig, redirectCode),
                     RedirectUrlsEntity.buildNokUrl(uiConfig, redirectCode),
-                    UUID.fromString(restRequestContext.getRequestId()),
+                    xRequestId,
+                    null,
```

Comment:
Please add a comment about why the values are null, or add some constant like:
`public static final LocalDateTime COMPUTE_X_TIMESTAMP_UTC = null`
and use it here instead of null

Reply:
3 constants were added, with comments regarding the meaning of null values